server/maintenance_requests.go

```diff
@@ -1,34 +1,27 @@
 // This file provides NewApp() with functions for maintenance requests
 package main
 
-import(
-	"github.com/gin-gonic/gin"
-	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
-	"time"
+import (
+	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"net/http"
-	"encoding/json"
-)
+	"time"
 
-/*type CreateRequest struct {
-	Request string `json:"request"`
-	UserSubmitted int `json:"id"`
-	DateSubmitted time.Time `json:"date"`
-}*/
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+	"github.com/gin-gonic/gin"
+)
 
 /* AllMaintenanceRequests returns all the maintenance_requests from the database
 *  AllMaintenanceRequests is a function of app */
-func (app *App) AllMaintenanceRequests(c *gin.Context) {
-	/* Get data using psql.AllMaintenanceRequests and pass in app.db for the
-	* connection */
-
-	/* Takes the data returned in the psql func and formats it to a valid format
-	* using the gin.H func. See if it you can just use a Struct */
-
-	/* Use the gin.context.JSON func to return a status and the data. See auth.go
-	* for an example */
+func (app *App) GetMaintenanceRequests(c *gin.Context) {
+	var req = app.db.AllMaintenanceRequests()
```

Comment:
Let's use a more descriptive name, like `maintenance_requests`.

Reply:
To contradict joseph a little I think this is ok, long variable names are kinda ugly imo lol. but maybe change it to something like `reqs` or `requests` at least

you can also declare variables with `:=`, like `reqs := app.db.AllMaintenanceRequests()`

Reply:
Go with what Mike says 👍

Reply:
idk why, but go doesn't like it if I have a lower-case method "requests", but it's fine with "Requests"

Reply:
what's the error if you use `requests`?

Reply:
`app.db.requests undefined (cannot refer to unexported field or method requests)`

Reply:
Ah, I think you misunderstood what line we were talking about. We're not talking about line 17, the name of the function. We're talking about line 18, the name of your variable. Can you change the name and the interface back to `AllMaintenanceRequests`?

Reply:
okay, I get what you meant, changed the variable name and changed the method name back to AllMaintenanceRequests